backend/routes/passwordRoute.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+router.post('/check-password', async(req, res) => {
+    const {password} = req.body;
+    const findPassword = await prisma.user.findUnique({
+        where: {password}
+    })
+
+    const exists = findPassword !== null
+    res.json({exists});
```

Comment:
want to send something more descriptive like a status and a message in the json to process the error / success in the frontend.

ex: res.status(...).json({message: ...})

Reply:
I did add a few more status messages for other errors that may occur for now, let me know if they become overlapping.